### PR TITLE
Improve some logging for large rate limits

### PR DESCRIPTION
**Pull request by MinnDevelopment**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [x] Internal code
- [ ] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

We should always log long rate limits on warn level, unless its also a global rate limit (which is already logged elsewhere).

### CODE REVIEW

File: src/main/java/net/dv8tion/jda/internal/requests/ratelimit/BotRateLimiter.java [hunk 1]
```diff
@@ -266,11 +266,12 @@ else if (cloudflare)
                     // Handle hard rate limit, pretty much just log that it happened
                     else
                     {
-                        boolean firstHit = hitRatelimit.add(baseRoute);
+                        boolean firstHit = retryAfter >= 6000 || hitRatelimit.add(baseRoute);
```
Comment (gpluscb) [inline src/main/java/net/dv8tion/jda/internal/requests/ratelimit/BotRateLimiter.java:269]: Should this be `<`? We want to log it as debug only if it's *not* long or we haven't hit it before if I understand correctly.

Comment (MinnDevelopment) [reply src/main/java/net/dv8tion/jda/internal/requests/ratelimit/BotRateLimiter.java:269]: Good catch, my mistake.
```suggestion
                        boolean firstHit = retryAfter < 6000 || hitRatelimit.add(baseRoute);
```

File: src/main/java/net/dv8tion/jda/internal/requests/ratelimit/BotRateLimiter.java [hunk 2]
```diff
@@ -266,11 +266,12 @@ else if (cloudflare)
                     // Handle hard rate limit, pretty much just log that it happened
                     else
                     {
-                        boolean firstHit = hitRatelimit.add(baseRoute);
+                        boolean firstHit = retryAfter < 6000 || hitRatelimit.add(baseRoute);
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/internal/requests/ratelimit/BotRateLimiter.java:269]:
```suggestion
                        boolean firstHit = retryAfter < 60000 || hitRatelimit.add(baseRoute);
```

File: src/main/java/net/dv8tion/jda/internal/requests/ratelimit/BotRateLimiter.java [hunk 3]
```diff
@@ -266,11 +266,12 @@ else if (cloudflare)
                     // Handle hard rate limit, pretty much just log that it happened
                     else
                     {
-                        boolean firstHit = hitRatelimit.add(baseRoute);
+                        boolean firstHit = retryAfter < 60000 || hitRatelimit.add(baseRoute);
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/internal/requests/ratelimit/BotRateLimiter.java:269]:
```suggestion
                        boolean firstHit = hitRatelimit.add(baseRoute) && retryAfter < 60000;
```

**Pull request merged**